### ISSUE TITLE
Allow the public to create issues on a private server

### ISSUE THREAD

**Issue by MightyPork** [go-gitea/gitea#4262]
I apologize if this is a duplicate, didn't find it in the tracker.

I have a self-hosted gitea where I intend to move my projects to. I don't want people creating accounts, organizations and repositories on my server, but I would like them to be able to open issues.

Maybe I missed some config option that would allow this (?) but if no, this would be a valuable improvement. (This could of course be solved by federation, but we aren't there yet)

- Gitea version: 1.3.2

**Comment by JamesGKent** [comment]
I just implemented this option in gogs, I believe the file that would need changing is https://github.com/go-gitea/gitea/blob/master/models/repo_list.go but the code is sufficiently different in gitea that I can't see at a glance what you'd need to change.
for reference my changes to gogs: https://github.com/gogs/gogs/pull/5234/files
and just to explain how it works in gogs, the repo would be shown in search or explore if any part of it is public (code, issues or wiki) but only the public parts of it would show when opened. so for your case the repo should be set to private and the issues set to public.

**Comment by MightyPork** [comment]
I'm not sure the proposal was clear enough - this was about creating issues without registration on "public" repos (for a gitea with registrations disabled). Alternatively to allow registration, but without the rights to create repositories - I don't want my small VPS to become another "github" or "gitlab.com".

**Comment by JamesGKent** [comment]
@MightyPork  sorry I misunderstood, I've just realised that what you want would require something like a passwordless guest account or similar. please ignore my comment.

**Comment by MightyPork** [comment]
Yeah or maybe "login with GitHub" or similar, creating an "external user" account without some permissions. I would consider trying to add this if I knew any Go.

For reference, here is my private server with the public project:
https://git.ondrovo.com/gex/gex-core

And a github mirror (which now has issues on it, as this is where the project started; but I would prefer the issues to live with the upstream):
https://github.com/gexpander/gex-core

**Comment by kolaente** [comment]
As a workaround, you could open your instance for registration but set org and repo limits to 0. That way people could create issues and such, but no repos or orgs.

See https://docs.gitea.io/en-us/config-cheat-sheet/ under "Repository"

**Comment by leggettc18** [comment]
Would that limit also apply to administrator users? Or only to general users? Is there a way to override those settings on a per-user basis? I also want to be able to do this without locking myself out from making more repos.

**Comment by kolaente** [comment]
You can set a per user limit for repos

**Comment by lunny** [comment]
I think https://github.com/go-gitea/gitea/pull/5833 will fix this.

**Comment by mckaygerhard** [comment]
hi +1 for this wanted feature, seems #5833 are a WIP for this get solved!

**Comment by lunny** [comment]
It seems this is a duplication of #639

**Comment by 6543** [comment]
> Yeah or maybe "login with GitHub" or similar, creating an "external user" account without some permissions. I would consider trying to add this if I knew any Go.
> 
> For reference, here is my private server with the public project:
> https://git.ondrovo.com/gex/gex-core
> 
> And a github mirror (which now has issues on it, as this is where the project started; but I would prefer the issues to live with the upstream):
> https://github.com/gexpander/gex-core

@MightyPork
 * "login with GitHub" is implemented :D
 * is it a dublicate of #639

so might considder closing this issue ...?

**Comment by Aviortheking** [comment]
I thinks it's more like bitbucket have that allow "anonymous" issues

**Comment by xeruf** [comment]
Yes, guest access, similar as mentioned in https://github.com/go-gitea/gitea/issues/8649#issuecomment-1204513734
